libs/perun/components/src/lib/entity-search-select/entity-search-select.component.html

```diff
@@ -1,4 +1,4 @@
-<mat-form-field class="w-100">
+<mat-form-field class="w-100 mt-3">
```

Comment:
with this change you did a change to all entity search selects which could very negatively impact the look on some of the search selects